src/Game.js

```diff
@@ -0,0 +1,137 @@
+function Game(){
+  this.score = 0;
+  this.scoreArray = [];
+  this.MAXPOINTS = 10;
+  this.bonusPoints = 0;
+  this.scoreFrame = 0;
+  this.bowlTurn = 0;
+  this.frame = 1;
+  this.turnScore = 0;
+  this.sparePoints = 0;
+  this.TOTALFRAME = 10;
+  this.strikeBowl = false;
+  this.spare = false;
+  this.lastFrameBonusPoints = 0;
+  this.lastFrameRols = 0;
+  this.bonusPointsCount = 0;
+}
+
+Game.prototype.calcScore = function(){
+  var total = 0;
+  for(var i in this.scoreArray) { total += this.scoreArray[i]; }
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.